client/errors_test.go

```diff
@@ -17,6 +17,7 @@ import (
 )
 
 func TestNewUnexpectedType(t *testing.T) {
+	t.Parallel()
```

Comment:
nitpick: This line very much appears to be useless, as it is the only such call within the package

Reply:
what about leaving it as a default norm to for the tests to be defined?

Reply:
I really dont think we should suggest or enforce that, it is essentially dead, untested, code.  It suggests that the test runs in parallel, but it does not, and in many cases the ability to run that test in parallel has/will not be tested. The introduction of a second Parallel call may cause the first/original test to fail.

Reply:
resolved

Reply:
*trivia*: in Rust tests run in parallel by default, but can be made sequential with a synchronization mechanism or `#[serial]` attribute from crate https://docs.rs/serial_test/

Reply:
Yes, I'm a bigger fan of this way round.  Opting out of running them in parallel is more like accepting (and warning/flagging) that the production code is broken and needs fixing.